tensorflow/core/kernels/sparse_matmul_op.h

```diff
@@ -18,6 +18,22 @@ limitations under the License.
 
 #include "third_party/eigen3/Eigen/Core"
 #include "tensorflow/core/platform/types.h"
+#ifdef _MSC_VER
```

Comment:
is it possible to find a better home for this, somewhere in platform? Maybe platform/windows/intrinsics_port.h?